pkg/agent/apiserver/handlers/bgppeer/handler.go

```diff
@@ -38,7 +38,14 @@ func HandleFunc(bq querier.AgentBGPPolicyInfoQuerier) http.HandlerFunc {
 			return
 		}
 
-		peers, err := bq.GetBGPPeerStatus(r.Context())
+		ipv4Peers := r.URL.Query().Has("ipv4")
```

Comment:
same comment as on the bgproutes PR: if `ipv4=false` is used it will include IPv4 peers. This is a pretty minor issue, since no one is expected to call this API save for antctl, but it's a bit surprising nonetheless. However, I see that we did the same thing in another existing command (`tableNamesOnly := r.URL.Query().Has("table-names-only")`).
Maybe a good compromise here which keeps things simple is to use: `ipv4Peers := r.URL.Query().Get("ipv4") == "true"`

Reply:
I agree there is a minor issue which should be solved.
I wanted the default output to list all ipv4 and ipv6 peers when no flag is passed.
So, now after solving this minor issue our output will be empty when no flag is passed, we make it kind of mandatory to pass atleast one of the two flags.

I think `ipv4Peers := r.URL.Query().Has("ipv4")` is better than `ipv4Peers := r.URL.Query().Get("ipv4") == "true"` as passing the flag means true and not passing it means false by default. We also document it that just pass `--ipv4` or/and `--ipv6`
If we use `ipv4Peers := r.URL.Query().Get("ipv4") == "true"`, we compel user to pass exactly `--ipv4=true` which is a larger string to type than just `--ipv4` only.
In the existing code, i can remove the condition which sets both flags to true when both are false. 
Does this sounds good ?

Reply:
The url in testcases are currently wrong, it should be just `?ipv4` and not `?ipv4=true` given that `.Has` has been used and not `.Get`

Reply:
I also think `ipv4=false` is wrong usage as it is a boolean flag and not a typical string flag with some value. We also document to use it just `--ipv4`

Reply:
> I also think ipv4=false is wrong usage as it is a boolean flag and not a typical string flag with some value.

You cannot say it's wrong usage if it is not enforced

> We also document to use just --ipv4

antctl is not the same as the API

Reply:
done, PTAL.
Thanks @antoninbas